### DISCUSSION TITLE
chapter15:TypeError: customChains is not iterable

### DISCUSSION THREAD

**Discussion by Bilalbasheer100** [graphprotocol/graph-tooling#1260] [General]
when iam verifying the contract this error pops up....my contract is deployed in sepolia test network but icant verify 
```javascript
hardhat.config.js
______________
etherscan: {        apiKey: {
            sepolia: ETHERSCAN_API_KEY,
            polygon: POLYGONSCAN_API_KEY,
        },
    },
```
```
Error
____

TypeError: customChains is not iterable
```

**Comment by Bilalbasheer100** [comment]
i have already used the methods specified in the discussions,unfortunately none of the actually works.

**Reply by saihaj** [reply]
Can you share a [minimal reproduction](https://en.wikipedia.org/wiki/Minimal_working_example). You can either use [StackBlitz](https://stackblitz.com/), [CodeSandbox](https://codesandbox.io/) or a public GitHub repo.

**Comment by azf20** [comment]
Hey @Bilalbasheer100 this looks like an issue with contract verification in Hardhat rather than relating to Graph Tooling (e.g. graph-cli)? Please re-ping if not
